var/spack/repos/builtin/packages/py-lxml/package.py

```diff
@@ -27,8 +28,9 @@ class PyLxml(PythonPackage):
 
     depends_on('python@2.7:2.8,3.5:', type=('build', 'run'))
     depends_on('py-setuptools', type='build')
-    depends_on('libxml2', type=('build', 'link', 'run'))
-    depends_on('libxslt', type=('build', 'link', 'run'))
+    depends_on('libxml2@2.9.2:', type=('build', 'link', 'run'))
+    depends_on('libxslt@1.1.28:', type=('build', 'link', 'run'))
```

Comment:
```suggestion
    depends_on('libxslt@1.1.27:', type=('build', 'link', 'run'))
```
Spack will still default to the latest version, but it sounds like 1.1.27 should still work even if it isn't recommended.